From a77a96070679661a2455decddc4a8df3db30e830 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Tue, 2 Jun 2020 21:51:08 +0800
Subject: [PATCH] ARM: dts: rockchip: rv11xx-evb-v10: fix vcc18_lcd regulator

vcc18_lcd is the power enable pin of dsi panel board, and it
also provides the rst signal of panel through RC circuit.
When we power on and change the pin directly, the rst will not
power down completely, resulting in the probability that the
screen cannot display after reboot.
so fix it.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: Ifb47ea23a4bd2d9e41fd317d3c7777e158e54dec
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 40f8ee391bc2..61fd73e2e23e 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -142,7 +142,6 @@
 	vcc18_lcd_n: vcc18-lcd-n {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc18_lcd_n";
-		regulator-boot-on;
 		gpio = <&gpio3 RK_PA7 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
 	};
-- 
2.35.3

